sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CodeGenerationSuite.scala

```diff
@@ -42,4 +42,8 @@ class CodeGenerationSuite extends SparkFunSuite {
 
     futures.foreach(Await.result(_, 10.seconds))
   }
+
+  test("SPARK-8443: code size limit") {
+    GenerateMutableProjection.generate(List.fill(5000)(EqualTo(Literal(1), Literal(1))))
```

Comment:
Will codegen cause exception if the code size is too large? Or we at least to execute the code once?